examples/express/index.ts

```diff
@@ -0,0 +1,135 @@
+import express, { Router } from 'express'
+import SgidClient from '@opengovsg/sgid-client'
+import * as dotenv from 'dotenv'
+import crypto from 'crypto'
+import cookieParser from 'cookie-parser'
+import { fetchStaticFiles } from './helpers'
+import open from 'open'
+
+dotenv.config()
+
+const PORT = 5000
```

Comment:
port 5000 has some [issues](https://developer.apple.com/forums/thread/682332) on macs (@kschiew and i faced this issue). i think we should move to a less problematic port haha